templates/arm/stm32f4cube/build.conf

```diff
@@ -1,3 +1,8 @@
+include $(ROOT_DIR)/mk/core/compiler.mk
+
+# to see which target clang supports:
```

Comment:
As this is a part of template, this checks supposed to replicate in all templates?
How about to create separate templates for clang?